flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/FlinkService.java

```diff
@@ -107,7 +108,11 @@ public void submitSessionCluster(FlinkDeployment deployment, Configuration conf)
         final int port = config.getInteger(RestOptions.PORT);
         final String host =
                 config.getString(
-                        RestOptions.ADDRESS, String.format("%s-rest.%s", clusterId, namespace));
+                        RestOptions.ADDRESS,
```

Comment:
Currently, the `RestOptions.ADDRESS` is always null in effective configuration. So we could simply always use the rest service name here. Right?

Reply:
Agree. I will follow your suggestion.